e2e/test/iothub/method/MethodFaultInjectionTests.cs

```diff
@@ -271,7 +287,7 @@ await FaultInjection
                     faultType,
                     reason,
                     FaultInjection.DefaultFaultDelay,
-                    FaultInjection.DefaultFaultDelay,
+                    FaultInjection.DefaultFaultDelay, // we want a quick one because we need time to recover
```

Comment:
Should one of these be ```DefaultFaultDuration```?

Reply:
No, this test chooses a smaller value for duration so it recovers in time. That's why I added the comment.

I too thought it was a bug and changed it, only to find it made the tests fail.

Reply:
I'm more pointing out that we are passing in "DefaultFaultDelay" into what I assume is a "FaultDuration" parameter here

Reply:
I'd be fine with a magic number here with the comment you left, but otherwise it _looks_ like a test bug

Reply:
Yep. I just commented what looked like a test bug to me, too.